src/main/java/com/smartbear/readyapi/client/teststeps/datasource/datagen/PhoneNumberDataGeneratorBuilder.java

```diff
@@ -0,0 +1,33 @@
+package com.smartbear.readyapi.client.teststeps.datasource.datagen;
+
+import com.smartbear.readyapi.client.model.DataGenerator;
+import com.smartbear.readyapi.client.model.PhoneNumberDataGenerator;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class PhoneNumberDataGeneratorBuilder extends AbstractDataGeneratorBuilder<PhoneNumberDataGeneratorBuilder> {
+    private final PhoneNumberDataGenerator phoneNumberDataGenerator = new PhoneNumberDataGenerator();
+
+    private final String numberFormatString = "XXX-XXX-XXXX,+1 XXX-XXX-XXXX,+1 (XXX)-XXX-XXXX,+X XXX-XXX-XXXX,+X (XXX)-XXX-XXXX";
+    private final List<String> allowedFormatsList = Arrays.asList(numberFormatString.split(","));
+
+    PhoneNumberDataGeneratorBuilder(String property) {
+        super(property);
+        phoneNumberDataGenerator.setType("Phone Number");
+        phoneNumberDataGenerator.setNumberFormat("XXX-XXX-XXXX");
+    }
+
+    public PhoneNumberDataGeneratorBuilder withNumberFormat(String numberFormat) {
```

Comment:
I don't know what to call these different format, so rather I created a method to provide a format. If we can name these formats then we can individual methods for setting these formats without asking user to provide the string value.